src/luascript.cpp

```diff
@@ -10463,6 +10541,52 @@ int LuaScriptInterface::luaHouseSetAccessList(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaHouseStartTrade(lua_State* L)
+{
+	// house:startTrade(player, tradePartner)
+	House* house = getUserdata<House>(L, 1);
+	Player* player = getUserdata<Player>(L, 2);
+	Player* tradePartner = getUserdata<Player>(L, 3);
+
+	if (!player || !tradePartner || !house) {
```

Comment:
Just because it's going to be checked in the sell house talkaction, it doesn't mean that someone could still mess this up in another place.